usecases/config/environment.go

```diff
@@ -395,6 +396,14 @@ func parseRAFTConfig(hostname string) (Raft, error) {
 		return cfg, err
 	}
 
+	if err := parsePositiveInt(
+		"RAFT_GRPC_MAX_SIZE",
```

Comment:
If you have an alternate name you think we should use for the env var, feel free to propose it here

Reply:
suggestion: `RAFT_GRPC_MESSAGE_MAX_SIZE`

Reply:
Thanks, just updated the PR with this name